integration/integration.test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+
+NUMBER=$RANDOM
```

Comment:
Where does `$RANDOM` get defined?

Reply:
`$RANDOM` is unix system native such as `$HOME`

Reply:
The `$RANDOM` is passed as `request.params` to server and the bash script checks if the same number passed is returned in the `response` json